deployment/src/main/java/io/quarkiverse/argocd/deployment/ScmInfoBuildItem.java

```diff
@@ -55,7 +55,7 @@ public static Optional<ScmInfoBuildItem> fromPath(Path path) {
             return scmInfo;
         }
         try {
-            Map<String, String> remotes = Git.getRemotes(path);
+            Map<String, String> remotes = io.dekorate.utils.Git.getRemotes(path);
```

Comment:
I think that this was meant to refer to our util. not dekorate, right?

Reply:
Yep but should we then import ALL the methods from Git util Dekorate class ?

Reply:
I will merge as is and proceed with changes later.